include/nicegraf.h

```diff
@@ -3482,6 +3554,31 @@ void ngf_renderdoc_capture_begin() NGF_NOEXCEPT;
  */
 void ngf_renderdoc_capture_end() NGF_NOEXCEPT;
 
+/**
```

Comment:
all of the changes to `nicegraf.h` should be discarded, since we will keep the non-platform specific portions of the API intact.